### ISSUE TITLE
Reconciler: Show a visual diff for each suggested match

### ISSUE THREAD

**Issue by zarino** [everypolitician/everypolitician#234]
It's hard to quickly scan every line of details about each suggested match, and compare them with the incoming person.

I have a feeling some sort of visual diff (like a Github change diff) would help highlight things that are the same (and/or things that are different).

**Comment by zarino** [comment]
I was thinking something like this:

![screen shot 2015-12-02 at 11 40 47](https://cloud.githubusercontent.com/assets/739624/11530145/18d1f042-98ea-11e5-95fb-803bedf98a4f.png)

In this case, all the suggestions are weak, so there's not a clear winner.

@tmtmtmtm – I wonder whether we also need to show which bits of text **don’t** match (maybe in red?) – but at the same time, I'm guessing we want to avoid having literally every bit of text in either red or green, because that'll be just as hard to parse. Thoughts?

**Comment by tmtmtmtm** [comment]
@zarino: Yep — this would be a huge win. I suspect a red-only version might be clearer than a green-only version, but it's hard to know for sure without trying it out for a while. But literally anything along these lines would be a significant improvement even if we decide to tweak later. (i.e. let's get a Simplest Thing version live ASAP, and evolve…)

**Comment by zarino** [comment]
FWIW here's how it'd look with the **differences** highlighted rather than the **similarities**:

![screen shot 2015-12-02 at 12 00 26](https://cloud.githubusercontent.com/assets/739624/11530419/5ccb8892-98ec-11e5-9fb4-4b84390a3499.png)

**Comment by tmtmtmtm** [comment]
Thanks. Hard to know which would be better — the red seems better at the level of a single screen like this, but as it's more likely that there'll be more differences than similarities it might get to be too much over time. I'm happy with either for now, as I assume it'll be fairly simple to switch to the other later.

**Comment by zarino** [comment]
It'll also partly depend on whatever library we use to do the diffing (assuming there's one out there that's suitable).

**Comment by tmtmtmtm** [comment]
I've done an initial version of this that's just red or green at a full field level (https://github.com/everypolitician/everypolitician-data/pull/1851). This is, as expected, a MASSIVE win (I was able to reconcile Israel with this in a fraction of the time it was taking previously). I'm not sure that doing anything more fine-grained will really be enough of a win, but we can revisit later if there's an obvious library available that gives us this…

**Comment by zarino** [comment]
@tmtmtmtm – looks good to me! If you think the full field diff is enough for now, feel free to close this ticket.
